### PR TITLE
don't forget about data quality

### PR DESCRIPTION
**Pull request by steveoh**
we should verify that sweeper and the metadata are returning no issues before adding the data to the sgid or anywhere. I think this is a good place to validate that. Thoughts? 

Feel free to tag others. I'm not sure wants to review these changes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ you can strike them out. They are available if for some reason the internal sgid
 - [ ] SGID10
 - [ ] ArcGIS Online
 
+<!-- 
+the data should be in good shape.
+-->
+# The data is high quality
+
+- [ ] Sweeper checks have run and passed
+- [ ] The minimum metadata fields are populated
```
Comment (stdavis) [inline .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md:51]:
```suggestion
```
[Sweeper already checks metadata](https://github.com/agrc/sweeper/blob/master/src/sweeper/sweepers/metadata.py). If it's missing something we should add it to sweeper.

Comment (gregbunce) [reply .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md:51]: great addition.  i'll post the metadata minimum requirements doc on the webpage.  also, we should also add "Comply with domain rules" and I can clean up that domain doc that lays out  our plan and rules.

Comment (gregbunce) [reply .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md:51]: @stdavis maybe we can change the metadata piece to remind the user of our minimum requirements.

[the remark this answers is not in the document]
Comment (gregbunce) [reply .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md:51]: either way, i'll update this [page](https://gis.utah.gov/about/policy/sgid/) to add the domain rules and also add a link from the metadata section to our more-detailed metadata doc.

[the remark this answers is not in the document]
Comment (stdavis) [reply .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md:51]: On second thought, I agree that it's best to put these here even though sweeper checks them. 👍

**Comment by gregbunce** [comment]
just rambling here, but i'll also, update the "Adding New Tables" section of [this page](https://gis.utah.gov/about/policy/sgid/) to the repo that's currently called cemetery.  have we decided on a new name yet?

File: .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (steveoh) [reply .github/ISSUE_TEMPLATE/introduce-sgid-dataset.md]: woops i accepted scott's change without reading this.

**Comment by steveoh** [comment]
> just rambling here, but i'll also, update the "Adding New Tables" section of [this page](https://gis.utah.gov/about/policy/sgid/) to the repo that's currently called cemetery. have we decided on a new name yet?

Yes lets put this into the policy for sure.

**Pull request merged**